packages/renderer/src/App.svelte

```diff
@@ -128,7 +129,10 @@ window.events?.receive('display-troubleshooting', () => {
             resourceId="{decodeURI(meta.params.resourceId)}"
             engineId="{decodeURI(meta.params.engineId)}" />
         </Route>
-        <Route path="/pods/:kind/:name/:engineId/*" breadcrumb="Pod Details" let:meta navigationHint="details">
```

Comment:
Did you forget to add the navigationHint in the Pod details route?

Reply:
Good catch, the new page should have it as well. You can check the breadcrumb (especially after selecting an action and cancelling it) to confirm correct behaviour.